data-sources/lambdatest/config.yml

```diff
@@ -0,0 +1,9 @@
+id: lambdatest
+displayName: LambdaTest
+description: |
+  LambdaTest is an AI-powered test orchestration and execution platform to run manual and automated tests at scale. With LambdaTest, developers and testers can test their websites and mobile applications across 3000+ real browsers, devices, and operating systems.
```

Comment:
@prateeksainilambdatest 

This has to be under 240 characters, it is currently 263. Can you condense this description a bit so it fits correctly in the tile?

Reply:
Updated